pkg/querier/frontend/frontend.go

```diff
@@ -180,7 +184,14 @@ func (f *Frontend) Handler() http.Handler {
 }
 
 func (f *Frontend) handle(w http.ResponseWriter, r *http.Request) {
+	startTime := time.Now()
 	resp, err := f.roundTripper.RoundTrip(r)
+	queryResponseTime := time.Now().Sub(startTime)
+
+	if queryResponseTime > f.cfg.LogQueriesLongerThan {
+		level.Info(f.log).Log("msg", "slow query", "url", fmt.Sprintf("http://%s/", r.URL.Host+r.URL.Path+r.URL.RawQuery), "time", queryResponseTime.String())
```

Comment:
A few comments on this line:
- Is INFO the correct level for the output? Considering this only outputs for queries that exceed a time threshold set by the operator to be notified about, perhaps this should be WARN.
- Please revisit the output for the `url` field. The string format template and the values concatenated do not appear to rebuild the original url as it was sent i, if that is the intention.
- `time` may not a be a good field name for a duration filed. Please consider `time-taken`
- The `orgID` the query was executed against may be a valuable piece of information to make this data actionable by anyone attempting to reproduce the query load on the backend store.

Reply:
Thanks @khaines For feedback. 

> Is INFO the correct level for the output? Considering this only outputs for queries that exceed a time threshold set by the operator to be notified about, perhaps this should be WARN.

Done.

> Please revisit the output for the url field. The string format template and the values concatenated do not appear to rebuild the original url as it was sent i, if that is the intention.

Updated, now using `rHost+r.RequestURI` to get the original request URL.

> time may not a be a good field name for a duration filed. Please consider time-taken

Done.

> The orgID the query was executed against may be a valuable piece of information to make this data actionable by anyone attempting to reproduce the query load on the backend store.

Done.